.travis.yml

```diff
@@ -1,26 +1,29 @@
 language: objective-c
-cache:
-  - bundler
-  - cocoapods
-osx_image: xcode7.3
-
-before_install:
-  - brew update || brew update
-  - brew outdated xctool || brew upgrade xctool
-  - SIMULATOR_ID=$(xcrun instruments -s | grep -o "iPhone 6 (9.3) \[.*\]" | grep -o "\[.*\]" | sed "s/^\[\(.*\)\]$/\1/")
+osx_image: xcode11.2
 
 install:
   - bundle install
-  - bundle exec pod install
+  - bundle exec pod repo update --silent
+  - travis_wait bundle exec pod install || { rm -rf ./Pods; bundle exec pod install; }
 
 script:
-  - echo $SIMULATOR_ID
-  # Launching the simulator before building and testing is necessary to prevent spurious timeouts.
-  # Specifying the simulator UDID is also necessary.
-  - open -a "Simulator" --args -CurrentDeviceUDID $SIMULATOR_ID
-  - travis_wait bundle exec rake spec:unit
-  - bundle exec rake spec:api:integration
+  - set -o pipefail
+  - xcodebuild test -workspace BraintreeDropIn.xcworkspace -scheme "UnitTests" -configuration Debug -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 8,OS=12.2' | xcpretty -c
 
-notifications:
-  email:
-    - team-ios@getbraintree.com
+#language: objective-c
+#cache:
+#  - bundler
+#  - cocoapods
+#osx_image: xcode11.2.1
+#
+#before_install:
+#  - brew update || brew update
+#
+#install:
+#  - bundle install
+#  - bundle exec pod repo update --silent
+#  - bundle exec pod install
+#
+#script:
+#  - travis_wait bundle exec rake spec
+#
```

Comment:
I'm a little confused why we left this commented out code? Can we add a comment to describe why it is there maybe?

Reply:
Oh shoot, we can probably revert this. Steven and I were trying to debug why Travis kept failing, but we never resolved it. 

As a side note, I was thinking we could try [GitHub Actions](https://github.com/features/actions) as an alternative to Travis. Could be a good project for Open Dev.

Reply:
Looks like travis build can't complete now

Reply:
Yeah, it hasn't been working for a while :(